pkg/ee/validation/machine/providers.go

```diff
@@ -76,6 +78,10 @@ const (
 	// Alibaba credential env.
 	envAlibabaAccessKeyID     = "ALIBABA_ACCESS_KEY_ID"
 	envAlibabaAccessKeySecret = "ALIBABA_ACCESS_KEY_SECRET"
+	// Packet credential env.
+	envPacketToken           = "PACKET_API_KEY"
+	envPacketProjectID       = "PACKET_PROJECT_ID"
+	packetCustomInstanctType = "storage.custom"
```

Comment:
```suggestion
	packetCustomInstanceType = "storage.custom"
```

Btw what is this? In this case we cant get the resource quantities?

Reply:
for this type, i got these details:
`packngo.Plan{ID:"d249fd31-0af2-4318-8393-1b130beadde7", Slug:"storage.custom", Name:"Custom Storage Appliance", Description:"Custom storage appliance", Line:"baremetal", Specs:packngo.Specs{Cpus:[packngo.Cpus{Count:2, Type:"PureStorage Control"}], Memory:packngo.Memory{Total:""}, Drives:[packngo.Drives{Count:0, Size:"", Type:""}], Nics:[packngo.Nics{Count:0, Type:""}]}, Pricing:packngo.Pricing{Hour:0, Month:0}, DeploymentTypes:[], Class:"storage.custom", AvailableIn:[], AvailableInMetros:[], Href:""}`
as you can see data is nil for this type:
`Drives:[packngo.Drives{Count:0, Size:"", Type:""}
packngo.Memory{Total:""}`

Reply:
well for drives, but you can still get CPU. Do we need to treat it in a special way? It will just go through and set Memory and Storage to 0?

Reply:
well for  drive.Size: "" we get this err:
`error parsing machine storage request to quantity and drive.Size quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$' `

Reply:
we can take the value of the CPU and then put checks in place for drive and memory.
but as it was a "custom" type, I thought to better skip it.

Reply:
Ah ok, the empty string is no good.

Then add a skip if its empty? Better to defend against that, they could add some new type that also has this issue. WDYT?

Reply:
yeah, I mean currently it doesn't have any other type with an empty string.
for I agree, we can be better prepared for future added types.
so for this type, we will only get CPU values, and i will put checks for drive and memory